### PR TITLE
BNFeth 138 50

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Updates

### CODE REVIEW

File: ParisBNF/et/BNFet50.xml [hunk 1]
```diff
@@ -42,85 +42,85 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
                         <title ref="LIT2697Sam" type="complete" xml:lang="gez">መጽሐፈ፡ ነገሥት፡ ቀዳማዊ።</title>
                         <textLang mainLang="gez"/>
-                        <colophon xml:lang="gez" xml:id="coloph1">ለመጽሐፈ፡ ነገሥት፡ ቀዳማዊ፡ ወጠኖ፡ 
+                        <colophon xml:lang="gez" xml:id="coloph1">ለመጽሐፈ፡ ነገሥት፡ ቀዳማዊ፡ ወጠኖ፡
                   <persName ref="PRS11010Samuel" role="scribe">ሳሙኤል</persName>፡
                   ወፈጸምዎ፡ <persName ref="PRS11011Natan" role="scribe">ናታን</persName>፡ ወ<persName ref="PRS11012Gad" role="scribe">ጋድ</persName>።</colophon>
-                        <note>See for the variants of the first chapter of <ref target="#ms_i2.1"/> 
-                           <ref target="#"/>.</note> 
+                        <note>See for the variants of the first chapter of <ref target="#ms_i2.1"/>
+                           <ref target="#"/>.</note>
```
Comment (eu-genia) [inline ParisBNF/et/BNFet50.xml:64]: why empty ref target? needed? can be added?

File: ParisBNF/et/BNFet138.xml [hunk 2]
```diff
@@ -30,73 +30,101 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                       <textLang mainLang="gez"/>
+                       <incipit xml:lang="gez"></incipit>
+                    </msItem>
+
+                  <msItem xml:id="ms_i6">
+                     <locus from="98va" to="106ra"/>
+                     <title type="complete" ref="LIT1929Mashaf">Maṣḥafa fǝlsatu la-Takla Hāymānot</title>
                      <textLang mainLang="gez"/>
+                     <note>The text starts with the story about the last days of Moses,
+                       <bibl><ptr target="bm:Nosnitsin2003Felsatu "/></bibl>.
```
Comment (eu-genia) [inline ParisBNF/et/BNFet138.xml:110]: remove space before` "/>`

File: ParisBNF/et/BNFet50.xml [hunk 3]
```diff
@@ -42,85 +42,85 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
                 which have been erased
                 and replaced by others. The corrected passages often have the same reading as <ref type="mss" corresp="BNFet7"/>,
                 but there are also several different readings in both manuscripts. </note>
-              
+
                      <msItem xml:id="ms_i3.1">
                         <locus from="59r" facs="f119"/>
                         <title ref="LIT3194Susanna" type="complete"/>
                         <textLang mainLang="gez"/>
                         <note>For examples of the variants of this manuscript, see <ref target="#"/>
```
Comment (eu-genia) [inline ParisBNF/et/BNFet50.xml:103]: ref?

File: ParisBNF/et/BNFet50.xml [hunk 4]
```diff
@@ -42,85 +42,85 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
-                        </note> 
+                        </note>
                         <!--add ref-->
                      </msItem>
-              
+
                      <msItem xml:id="ms_i3.2">
                         <title ref="LIT1259Daniel" type="complete"/>
                         <textLang mainLang="gez"/>
                         <note>For examples of the variants of  Daniel 7, see <ref target="#"/>
```
Comment (eu-genia) [inline ParisBNF/et/BNFet50.xml:111]: ref?

File: ParisBNF/et/BNFet138.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/et/BNFet138.xml:110]: ok

File: ParisBNF/et/BNFet50.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/et/BNFet50.xml:64]: Concerning this and a few more corrections below (I could have told this in the beginning), I revised only the segment with the genealogy and around. I did not do anything else on this item.

Comment (eu-genia) [reply ParisBNF/et/BNFet50.xml:64]: @thea-m is it ok to approve and merge or do you think the empty targets can be removed or filled? thank you!

**Comment by thea-m** [comment]
Updated the record (we were very ambitious at the beginning...)

**Review by eu-genia — APPROVED** [review]

**Pull request merged**